Unix/samples/Providers/TestClass_AllDMTFArrays/TestClass_AllDMTFArrays.cpp

```diff
@@ -119,14 +120,15 @@ void SetUint32Array(TestClass_AllDMTFArrays *instance, MI_Uint32 count)
     {
         for(MI_Uint32 i=0; i < count; i++)
         {
-            uint32Array[i] =  (MI_Uint32) i % 4294967296;
+            uint32Array[i] =  (MI_Uint32) i % 0xFFFFFFFF;
```

Comment:
Include file <limits.h> can be helpful here. You'd have `LONG_MAX`, `ULONG_MAX`, etc.

The problem with the original code is you didn't have `L` or `UL` decorations for data types of the constants. The definitions in <limits.h> have that.

This is fine is as, just pointing it out. I think the constants in <limits.h> would be more clear, but I won't hold up this review just for that.